performance/README.md

```diff
@@ -25,7 +25,7 @@ in `jq`. If there's glaring functional omissions that are limiting your use of
 `zq`, we welcome [contributions](../README.md#contributing).
 
 * For the permutations with `ndjson` input the recommended approach for
-[shaping Zeek NDJSON](https://zed.brimdata.io/docs/next/integrations/zeek/shaping-zeek-ndjson)
+[shaping Zeek NDJSON](https://zed.brimdata.io/docs/integrations/zeek/shaping-zeek-ndjson)
```

Comment:
This change was expected. When I made the changes in #4694 I knew the Zeek docs were going to be gone from their prior home on `main` in the repo, but they weren't yet revealed under any tagged version in the proper docs site, so for the moment I had to point to `next`. Now we have the tagged v1.9.0 docs, so that means I can drop the `next`.